epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyViewHolder.java

```diff
@@ -34,7 +34,7 @@ public void bind(@SuppressWarnings("rawtypes") EpoxyModel model,
 
     if (previouslyBoundModel != null) {
       // noinspection unchecked
-      model.bind(objectToBind(), previouslyBoundModel);
+      model.bindChanges(objectToBind(), previouslyBoundModel);
```

Comment:
I think there is a problem with calling `bindChanges` here - it will always be called, whether or not the model is reusing the view holder. If they are _not_ reusing the viewholder then they need to do a full bind, not just bind changes.

We could check if the model is reusing a view holder and do `bindChanges` or just `bind` depending on that.